js/index.js

```diff
@@ -1 +1,192 @@
+
+
+// my tools are
```

Comment:
good job bringing this in as  comment to make referencing it faster/easier on yourself.